cumulus/primitives/core/src/lib.rs

```diff
@@ -224,46 +246,72 @@ pub enum RelayBlockIdentifier {
 }
 
 /// Consensus header digests for Cumulus parachains.
-#[derive(Clone, RuntimeDebug, Decode, Encode, PartialEq)]
+#[derive(Clone, Debug, Decode, Encode, PartialEq)]
 pub enum CumulusDigestItem {
 	/// A digest item indicating the relay-parent a parachain block was built against.
 	#[codec(index = 0)]
 	RelayParent(relay_chain::Hash),
-	/// A digest item indicating which core to select on the relay chain for this block.
+	/// A digest item providing information about the core selected on the relay chain for this
+	/// block.
 	#[codec(index = 1)]
-	SelectCore {
-		/// The selector that determines the actual core.
-		selector: CoreSelector,
-		/// The claim queue offset that determines how far "into the future" the core is selected.
-		claim_queue_offset: ClaimQueueOffset,
-	},
+	CoreInfo(CoreInfo),
```

Comment:
I looked at potentially breaking changes related to the new digest and it seems that we should be fine, older collators will care about the RP and will iterate all digest items until they find the RP digest ([extract_relay_parent](https://github.com/paritytech/polkadot-sdk/blob/9969e1e81c94f2153412d647d92ecad8db3ccbf8/cumulus/primitives/core/src/lib.rs#L301))